Barotrauma/BarotraumaShared/SharedSource/Items/Components/Signal/ArithmeticComponent.cs

```diff
@@ -67,23 +67,24 @@ sealed public override void Update(float deltaTime, Camera cam)
             float output = Calculate(receivedSignal[0], receivedSignal[1]);
             if (MathUtils.IsValid(output))
             {
-                item.SendSignal(0, MathHelper.Clamp(output, ClampMin, ClampMax).ToString("G", CultureInfo.InvariantCulture), "signal_out", null);
+                item.SendSignal(MathHelper.Clamp(output, ClampMin, ClampMax).ToString("G", CultureInfo.InvariantCulture), "signal_out");
+                item.SendSignal(null, "");
```

Comment:
Is this supposed to be here?

Reply:
Not sure how that got there. Fixed in ce4d2a297a90962969e167faf5ae58f7597ce9f4.